standalone-integration/src/test/java/com/ericsson/bss/cassandra/ecchronos/standalone/ITOnDemandRepairJob.java

```diff
@@ -186,7 +184,7 @@ public void clean()
         reset(mockTableRepairMetrics);
     }
 
-    @AfterClass
+    @Parameterized.AfterParam
```

Comment:
What is the effect of this change?

Reply:
Things are initialized in method above with "@beforeparam", that's why we should have "afterparam" here otherwise we will leak resources each time beforeparam is called.